openstf/model/confidence_interval_applicator.py

```diff
@@ -137,3 +147,24 @@ def _add_quantiles_to_forecast(
             )
 
         return forecast
+
+    def _add_quantiles_to_forecast_quantile_regression(
```

Comment:
Good to add some docstring. A couple small suggestions, add new lines, indent and specify the quantiles data type

```python
        """Add quantiles to forecast.

            Use the standard deviation to calculate the quantiles.

        Args:
            forecast (pd.DataFrame): Forecast (should contain a 'forecast' + 'stdev' column)
            quantiles (List[float]): List with desired quantiles

        Returns:
            (pd.DataFrame): Forecast DataFrame with quantile (e.g. 'quantile_PXX')
                columns added.
        """
```